docs/articles/configuration/hocon.md

```diff
@@ -255,7 +255,7 @@ If a key has been specified more than once, the substitution will always evaluat
 
 If a substitution does not match any value present in the configuration and is not resolved by an external source, then it is undefined. An undefined _required substitution_ is invalid and will generate an error.
```

Comment:
Need to remove "and is not resolved by an external source,"

Reply:
Done